tests/test_builders/test_build_linkcheck.py

```diff
@@ -254,8 +254,8 @@ def test_raw_node(app):
         # ..and then the linkchecker will check that the root URL is available.
         index = (app.srcdir / "index.rst")
         index.write_text(
-            ".. raw:: html\n"
-            f"  :url: http://localhost:{port}/\n",
+            f".. raw:: {'html'}\n"
```

Comment:
Even if it's for review, it's better not to introduce unneeded f-strings. If you are worried about indentation visual, you can put an 'r' prefix.

Reply:
Perhaps using `str.format` for the second line could be neater; in that case neither of them is an f-string (it is indeed indentation alignment that I'd like to maintain here).

Reply:
Just use '%' instead. The performance between '%', 'format' and f-string is marginal (and we are testing) for such small strings.

Reply:
That would work, yep.  I think I prefer the results when using `str.format` and a keyword-based argument:

```python
index.write_text(
    ".. raw:: 'html'\n"
    "   :url: http://{netloc}/".format(netloc=netloc),
)
```

This would be the one exception where an f-string isn't suitable.